### DISCUSSION TITLE
Didn't work after reinstall SDwebUI (DirectML)

### DISCUSSION THREAD

**Discussion by songib** [ilian6806/stable-diffusion-webui-state#46] [Q&A]
I already deleted the Venv folder and extension aswell and reinstalled them and deleted the param aswell and it didn't work. 
what should I do?

**Comment by songib** [comment]
Seems like these settings cause a problem. ` [info] Options in main UI (setting entries that also appear in txt2img/img2img interfaces) (requires restart)`
![Stable Diffusion2023 07 27](https://github.com/ilian6806/stable-diffusion-webui-state/assets/8655587/99ddf265-e5f7-47c6-bcf6-c6574b474154)

**Comment by ilian6806** [comment]
I cant reproduce this with the latest version of web UI.
